fvm/context.go

```diff
@@ -13,10 +13,12 @@ import (
 
 // A Context defines a set of execution parameters used by the virtual machine.
 type Context struct {
-	Chain                        flow.Chain
-	Blocks                       Blocks
-	Metrics                      handler.MetricsReporter
-	Tracer                       module.Tracer
+	Chain   flow.Chain
+	Blocks  Blocks
+	Metrics handler.MetricsReporter
+	Tracer  module.Tracer
+	// LoadContextFromState is a flag telling the fvm to load certain parts of the context from the state
+	LoadContextFromState         bool
```

Comment:
I don't know of a better name, but is "loading the context" the right term? 
Maybe it can be more specific to what it currently configures, i.e. if the the execution parameters are set?

Reply:
Maybe we can name it `AllowContextOverrideByExecutionState`

Reply:
Renamed [43aa0ae](https://github.com/onflow/flow-go/pull/2678/commits/43aa0aeb5a4b56a5ca191c4314edd75124e89650)